src/main/webapp/app/overview/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, ElementRef, EventEmitter, Output, ViewChild } from '@angular/core';
```

Comment:
Client style warnings for unused imports. Please fix

Reply:
fixed